docs/BreakingChanges/112 Calling Attribute.GetCustomAttributes on an indexe.md

```diff
@@ -0,0 +1,45 @@
+## 112: Calling Attribute.GetCustomAttributes on an indexer property no longer throws AmbiguousMatchException if the ambiguity can be resolved by index's type
```

Comment:
{
  "Id": "112",
  "Title": "Calling Attribute.GetCustomAttributes on an indexer property no longer throws AmbiguousMatchException if the ambiguity can be resolved by index's type",
  "Details": "Prior to the .NET Framework 4.6, calling `GetCustomAttribute(s)` on an indexer property which differed from another property only by the type of the index would result in an `AmbiguousMatchException`. Beginning in the .NET Framework 4.6, the property's attributes will be correctly returned.",
  "Suggestion": "Be aware that GetCustomAttribute(s) will work more frequently now. If an app was previously relying on the `AmbiguousMatchException`, reflection should now be used to explicitly look for multiple indexers, instead.",
  "Link": null,
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Attribute.GetCustomAttribute(System.Reflection.MemberInfo,System.Type)",
    "M:System.Attribute.GetCustomAttribute(System.Reflection.MemberInfo,System.Type,System.Boolean)",
    "M:System.Attribute.GetCustomAttributes(System.Reflection.MemberInfo)",
    "M:System.Attribute.GetCustomAttributes(System.Reflection.MemberInfo,System.Boolean)",
    "M:System.Attribute.GetCustomAttributes(System.Reflection.MemberInfo,System.Type)",
    "M:System.Attribute.GetCustomAttributes(System.Reflection.MemberInfo,System.Type,System.Boolean)",
    "M:System.Reflection.CustomAttributeExtensions.GetCustomAttribute(System.Reflection.MemberInfo,System.Type)",
    "M:System.Reflection.CustomAttributeExtensions.GetCustomAttribute(System.Reflection.MemberInfo,System.Type,System.Boolean)",
    "M:System.Reflection.CustomAttributeExtensions.GetCustomAttribute``1(System.Reflection.MemberInfo)",
    "M:System.Reflection.CustomAttributeExtensions.GetCustomAttribute``1(System.Reflection.MemberInfo,System.Boolean)",
    "M:System.Reflection.CustomAttributeExtensions.GetCustomAttributes(System.Reflection.MemberInfo)",
    "M:System.Reflection.CustomAttributeExtensions.GetCustomAttributes(System.Reflection.MemberInfo,System.Boolean)",
    "M:System.Reflection.CustomAttributeExtensions.GetCustomAttributes(System.Reflection.MemberInfo,System.Type)",
    "M:System.Reflection.CustomAttributeExtensions.GetCustomAttributes(System.Reflection.MemberInfo,System.Type,System.Boolean)",
    "M:System.Reflection.CustomAttributeExtensions.GetCustomAttributes``1(System.Reflection.MemberInfo)",
    "M:System.Reflection.CustomAttributeExtensions.GetCustomAttributes``1(System.Reflection.MemberInfo,System.Boolean)"
  ],
  "Related": null,
  "VersionBroken": "4.6",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/976721",
  "Notes": "An analyzer could determine whether the property specified will be ambiguous or not (it usually won't be)\nSource analyzer status: Pri 3, Planned - No tracking work item yet",
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Attribute.GetCustomAttribute(System.Reflection.MemberInfo,System.Type)
- [x] M:System.Attribute.GetCustomAttribute(System.Reflection.MemberInfo,System.Type,System.Boolean)
- [x] M:System.Attribute.GetCustomAttributes(System.Reflection.MemberInfo)
- [x] M:System.Attribute.GetCustomAttributes(System.Reflection.MemberInfo,System.Boolean)
- [x] M:System.Attribute.GetCustomAttributes(System.Reflection.MemberInfo,System.Type)
- [x] M:System.Attribute.GetCustomAttributes(System.Reflection.MemberInfo,System.Type,System.Boolean)
- [x] M:System.Reflection.CustomAttributeExtensions.GetCustomAttribute(System.Reflection.MemberInfo,System.Type)
- [x] M:System.Reflection.CustomAttributeExtensions.GetCustomAttribute(System.Reflection.MemberInfo,System.Type,System.Boolean)
- [x] M:System.Reflection.CustomAttributeExtensions.GetCustomAttribute``1(System.Reflection.MemberInfo)
- [x] M:System.Reflection.CustomAttributeExtensions.GetCustomAttribute``1(System.Reflection.MemberInfo,System.Boolean)
- [x] M:System.Reflection.CustomAttributeExtensions.GetCustomAttributes(System.Reflection.MemberInfo)
- [x] M:System.Reflection.CustomAttributeExtensions.GetCustomAttributes(System.Reflection.MemberInfo,System.Boolean)
- [x] M:System.Reflection.CustomAttributeExtensions.GetCustomAttributes(System.Reflection.MemberInfo,System.Type)
- [x] M:System.Reflection.CustomAttributeExtensions.GetCustomAttributes(System.Reflection.MemberInfo,System.Type,System.Boolean)
- [x] M:System.Reflection.CustomAttributeExtensions.GetCustomAttributes``1(System.Reflection.MemberInfo)
- [x] M:System.Reflection.CustomAttributeExtensions.GetCustomAttributes``1(System.Reflection.MemberInfo,System.Boolean)